### ISSUE TITLE
mkdocs 1.0.1 via python 3.5 crash when attempting to use this variation of rtd theme

### ISSUE THREAD

**Issue by Azbests** [cjsheets/mkdocs-rtd-dropdown#19]
Theme does not seem to work with latest mkdocs that are installed from pip3.

```md
site_name: My Docs
theme:
    name: 'rtd-dropdown'
  
pages:
    - Home: index.md
```  

```bash
Requirement already up-to-date: mkdocs-rtd-dropdown in /usr/local/lib/python3.5/dist-packages (0.0.11)                                                                                                                 
Requirement already satisfied, skipping upgrade: mkdocs in /usr/local/lib/python3.5/dist-packages (from mkdocs-rtd-dropdown) (1.0.1)                                                                                   
Requirement already satisfied, skipping upgrade: Jinja2>=2.7.1 in /usr/local/lib/python3.5/dist-packages (from mkdocs->mkdocs-rtd-dropdown) (2.10)                                                                     
Requirement already satisfied, skipping upgrade: PyYAML>=3.10 in /usr/local/lib/python3.5/dist-packages (from mkdocs->mkdocs-rtd-dropdown) (3.13)                                                                      
Requirement already satisfied, skipping upgrade: livereload>=2.5.1 in /usr/local/lib/python3.5/dist-packages (from mkdocs->mkdocs-rtd-dropdown) (2.5.2)                                                                
Requirement already satisfied, skipping upgrade: click>=3.3 in /usr/local/lib/python3.5/dist-packages (from mkdocs->mkdocs-rtd-dropdown) (6.7)                                                                         
Requirement already satisfied, skipping upgrade: Markdown>=2.3.1 in /usr/local/lib/python3.5/dist-packages (from mkdocs->mkdocs-rtd-dropdown) (2.6.11)                                                                 
Requirement already satisfied, skipping upgrade: tornado>=5.0 in /usr/local/lib/python3.5/dist-packages (from mkdocs->mkdocs-rtd-dropdown) (5.1)                                                                       
Requirement already satisfied, skipping upgrade: MarkupSafe>=0.23 in /usr/local/lib/python3.5/dist-packages (from Jinja2>=2.7.1->mkdocs->mkdocs-rtd-dropdown) (1.0)                                                    
Requirement already satisfied, skipping upgrade: six in /usr/lib/python3/dist-packages (from livereload>=2.5.1->mkdocs->mkdocs-rtd-dropdown) (1.10.0) 
```

```bash
krisjanis@HAL-3000:~/Data/Temp/docs.test/derp$ mkdocs serve                                                                                                                                                            
INFO    -  Building documentation...                                                                                                                                                                                   
INFO    -  Cleaning site directory                                                                                                                                                                                     
Traceback (most recent call last):                                                                                                                                                                                     
  File "/usr/local/bin/mkdocs", line 11, in <module>                                                                                                                                                                   
    sys.exit(cli())                                                                                                                                                                                                    
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 722, in __call__                                                                                                                                   
    return self.main(*args, **kwargs)                                                                                                                                                                                  
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 697, in main                                                                                                                                       
    rv = self.invoke(ctx)                                                                                                                                                                                              
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 1066, in invoke                                                                                                                                    
    return _process_result(sub_ctx.command.invoke(sub_ctx))                                                                                                                                                            
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 895, in invoke                                                                                                                                     
    return ctx.invoke(self.callback, **ctx.params)                                                                                                                                                                     
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 535, in invoke                                                                                                                                     
    return callback(*args, **kwargs)                                                                                                                                                                                   
  File "/usr/local/lib/python3.5/dist-packages/mkdocs/__main__.py", line 134, in serve_command                                                                                                                         
    livereload=livereload                                                                                                                                                                                              
  File "/usr/local/lib/python3.5/dist-packages/mkdocs/commands/serve.py", line 119, in serve                                                                                                                           
    config = builder()                                                                                                                                                                                                 
  File "/usr/local/lib/python3.5/dist-packages/mkdocs/commands/serve.py", line 114, in builder                                                                                                                         
    build(config, live_server=live_server, dirty=dirty)                                                                                                                                                                
  File "/usr/local/lib/python3.5/dist-packages/mkdocs/commands/build.py", line 285, in build                                                                                                                           
    _build_theme_template(template, env, files, config, nav)                                                                                                                                                           
  File "/usr/local/lib/python3.5/dist-packages/mkdocs/commands/build.py", line 114, in _build_theme_template                                                                                                           
    output = _build_template(template_name, template, files, config, nav)                                                                                                                                              
  File "/usr/local/lib/python3.5/dist-packages/mkdocs/commands/build.py", line 93, in _build_template                                                                                                                  
    output = template.render(context)                                                                                                                                                                                  
  File "/usr/local/lib/python3.5/dist-packages/jinja2/environment.py", line 1008, in render                                                                                                                            
    return self.environment.handle_exception(exc_info, True)                                                                                                                                                           
  File "/usr/local/lib/python3.5/dist-packages/jinja2/environment.py", line 780, in handle_exception                                                                                                                   
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.5/dist-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/usr/local/lib/python3.5/dist-packages/rtd_dropdown/search.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/usr/local/lib/python3.5/dist-packages/rtd_dropdown/base.html", line 78, in top-level template code
    {%- block site_nav %}
  File "/usr/local/lib/python3.5/dist-packages/rtd_dropdown/base.html", line 79, in block "site_nav"
    {% include 'nav.html' %}
  File "/usr/local/lib/python3.5/dist-packages/rtd_dropdown/nav.html", line 11, in top-level template code
    {%- if nav_item_children[0].input_path is defined %}
  File "/usr/local/lib/python3.5/dist-packages/jinja2/environment.py", line 430, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: None has no element 0
```

**Comment by cjsheets** [comment]
I confirmed this is an issue and it appears related to the mkdocs V1 release.

I'm working on a fix and will bump the theme version to 1.0.0 once it's done. I don't anticipate the change taking too long. Thanks for the report.

**Comment by cjsheets** [comment]
Fixed with theme release 1.0.2
